config/log_rules.json

```diff
@@ -307,6 +307,42 @@
         "start_pattern": "Textures\\s+\\d+\\.?\\d* .b\\s+\\d{1,3}\\.?\\d*%",
         "end_pattern": "Complete size\\s+\\d+\\.?\\d* .b\\s+\\d{1,3}\\.?\\d*%",
         "store_lines": true
+      },
+      "command_invokation_failure": {
+        "active": true,
+        "start_pattern": "CommandInvokationFailure: (?<message>.+\\.)",
+        "store_lines": true,
+        "end_pattern": "^\\n",
+        "start_message": "[ANDROID] Command invokation failure: %{message}",
```

Comment:
Maybe we want to fix their typo in our message :) Or not. Same for the rules names.

WDYT?

Reply:
You mean invocation rather than invokation right? That would probably be clearer yeah. I was wondering if that's not an intended typo carrying some form of meaning, though I couldn't find any. I'll rename those